Relativity.Shared.SDK.md

```diff
@@ -0,0 +1,17 @@
+# Relativity.Shared.SDK Version Compatibility
+
+![Nuget](https://img.shields.io/nuget/v/Relativity.Shared.SDK)

```

Comment:
Mistake on my sample can you change this to - 
[![nuget](https://img.shields.io/nuget/v/Relativity.Shared.SDK.svg)](https://www.nuget.org/packages/Relativity.Shared.SDK)